BoardConfigCommon.mk

```diff
@@ -60,7 +62,6 @@ BOARD_KERNEL_CMDLINE := \
     msm_rtb.filter=0x3F \
     ehci-hcd.park=3 \
     androidboot.bootdevice=msm_sdcc.1
-BOARD_MKBOOTIMG_ARGS := --ramdisk_offset 0x02000000
```

Comment:
That flag is supported in the original boot image generation makefile too, it has nothing to do with the custom one